examples/0.6/BRUX00BEL.xml

```diff
@@ -1220,37 +1219,37 @@ All dimensions are in meters.]]></geo:insertTextGraphicFromAntenna>
         <geo:DOI codeSpace="urn:ga-gov-au:self.moreInformation-type"><![CDATA[https://doi.org/10.24414/ROB-GNSS-BRUX00BEL]]></geo:DOI>
       </geo:MoreInformation>
     </geo:moreInformation>
-  </geo:siteLog>
-  <geo:Document gml:id="site_picture_1">
-    <geo:type>image/jpeg</geo:type>
-    <geo:keywords>
-        <gmd:MD_Keywords>
+    <geo:associatedDocument>
+      <geo:Document gml:id="site_picture_1">
```

Comment:
This looks good now, `siteLog/associatedDocument/Document`. It is what we do at GA. However, we wouldn't also link to the site picture from the antenna element. Our site pictures are associated only with site log elements.